src/test/scala/uk/gov/hmrc/perftests/disareturns/InitialiseReturnsSubmissionRequests.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.perftests.disareturns
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.http.request.builder.HttpRequestBuilder
+import play.api.libs.json.Json
+import uk.gov.hmrc.performance.conf.ServicesConfiguration
+import uk.gov.hmrc.perftests.disareturns.models.InitialiseReturnsSubmissionPayload
+
+object InitialiseReturnsSubmissionRequests extends ServicesConfiguration {
+  val disaReturnsStubHost: String         = baseUrlFor("disa-returns-stub")
+  val reportingWindowPath: String         = "/test-only/setup-obligation-window"
+  val disaReturnsBaseUrl: String          = baseUrlFor("disa-returns")
+  val disaReturnsPath: String             = "/monthly/"
+  val initialiseReturnsSubmissionApiRoute = "/init"
+  val reportingWindowPayload: String      =
+    Json.stringify(Json.obj("reportingWindowOpen" -> true))
+
+  val initialiseReturnsSubmissionHeaders: Map[String, String] = Map(
+    "X-Client-ID"   -> "#{clientId}",
+    "Authorization" -> "#{bearerToken}"
+  )
+
+  val reportingWindowHeaders: Map[String, String] = Map(
+    "Content-Type" -> "application/json"
+  )
+
+  def getInitialiseReturnsSubmissionPayload(currentTaxYear: Int): InitialiseReturnsSubmissionPayload =
+    InitialiseReturnsSubmissionPayload(100, "APR", currentTaxYear)
+
+  val setReportingWindowsOpen: HttpRequestBuilder =
+    http("Set reporting window as Open")
+      .post(disaReturnsStubHost + reportingWindowPath)
+      .headers(reportingWindowHeaders)
+      .body(StringBody(reportingWindowPayload))
+      .asJson
+      .check(status.is(204))
+      .silent
+
+  val submitInitialiseReturnsSubmission: HttpRequestBuilder =
+    http("Submit 'initialise returns submission'")
+      .post(disaReturnsBaseUrl + disaReturnsPath + "#{isaManagerReference}" + initialiseReturnsSubmissionApiRoute)
+      .disableFollowRedirect
+      .headers(initialiseReturnsSubmissionHeaders)
+      .body(StringBody { session =>
+        val currentYear = java.time.Year.now().getValue.toInt
+        val payload     = getInitialiseReturnsSubmissionPayload(currentYear)
+        Json.stringify(Json.toJson(payload))
+      })
+      .asJson
```

Comment:
If you don't force a StringBody you don't need to stringify and then asJson again